BunnyInputComponent.cpp

```diff
@@ -190,8 +190,8 @@ void BunnyInputComponent::control(InputStruct input) {
         x_ = 0;
     }
 
-    if (input.right == false && input.left == false) {
-        x_ *= 0.5;
+    if (input.right == input.left) {
+        x_ *= 0.25;
```

Comment:
im pretty sure this makes them slow down more quickly. is that what you wanted?
as an example
factor = 0.5
`>>> [round(0.5**i * 10,3) for i in range(10)]`
`[10.0, 5.0, 2.5, 1.25, 0.625, 0.312, 0.156, 0.078, 0.039, 0.02]`
factor = 0.25
`>>> [round(0.25**i * 10,3) for i in range(10)]`
`[10.0, 2.5, 0.625, 0.156, 0.039, 0.01, 0.002, 0.001, 0.0, 0.0]`
what i suggested before was 0.95 or something generally closer to 1
`>>> [round(0.95**i * 10,3) for i in range(10)]`
`[10.0, 9.5, 9.025, 8.574, 8.145, 7.738, 7.351, 6.983, 6.634, 6.302]`

Reply:
Yes. It was intentional but I'll try those other values tomorrow. Not sure how it should feel. We will probably need to do some play testing